### PR TITLE
[#53] Refactored orderbook storage to redux store

### PR DESCRIPTION
**Pull request by alvyn279**
Addressed: 
- Moved away from props passing to a global store for `orderBook`

```js
// Redux STORE:
orderBook: {
    listItems: {
        "103.4": {
	    price: 103.4,
	    orders: [...],
	    type: 'ask',
	    isMiddle: 'false',
        }, 
        "103.39": {...},
    },
    maxQuantity: 20900
}

```

- TODO: fix all tests
- TODO: function test with increment of a single message

### CODE REVIEW

File: app/src/components/OrderBookSnapshot.js [hunk 1]
```diff
@@ -27,69 +30,67 @@ class OrderBookSnapshot extends Component {
...
+    handleChangeTime = (event, value) => {
+        let selectedTimeNano = parseInt(value);
+
+        this.setState({
+            selectedTimeNano: selectedTimeNano,
+            selectedTimeString: nanosecondsToString(selectedTimeNano),
+        });
+    };
+
+    /**
```
Comment (chris-baur) [inline app/src/components/OrderBookSnapshot.js]: why is all this considered different ?

File: app/src/components/PriceLevel.js [hunk 2]
```diff
@@ -6,26 +6,83 @@ import {Styles} from '../styles/PriceLevel';
 
 import Order from './Order';
 import {Box} from '@material-ui/core';
+import {connect} from 'react-redux';
+
+const MIN_PERCENTAGE_FACTOR_FOR_BOX_SPACE = 0.25;
```
Comment (chris-baur) [inline app/src/components/PriceLevel.js]: we currently have a constants file, I believe it should be moved to there unless that file is only for constants we plan on reusing

File: app/src/components/OrderBookSnapshot.js (hunk 1 above)
Comment (panoreak) [reply app/src/components/OrderBookSnapshot.js]: I think this is reverting the line endings. I just opened my pr, and I have a huge diff like this too.

**Comment by alvyn279** [comment]
After even further discussions, we will maintain a pass by props flow, except try to stop the rendering of the child components (PriceLevel) unless they are updated.

**Comment by alvyn279** [comment]
Will open a new PR addressing more up-to-date changes needed

**Pull request closed without merging**